main.py

```diff
@@ -0,0 +1,19 @@
+# Download the helper library from https://www.twilio.com/docs/python/install
+import os
+from twilio.rest import Client
+
+
+# Find your Account SID and Auth Token at twilio.com/console
+# and set the environment variables. See http://twil.io/secure
+account_sid = os.environ['ACCOUNT_SID']
```

Comment:
Can we get this data from secrets manager or vault instead of reading from env? That's safer for everyone